prerender_rails.gemspec

```diff
@@ -2,7 +2,7 @@
 
 Gem::Specification.new do |spec|
   spec.name          = "prerender_rails"
-  spec.version       = "1.7.0"
+  spec.version       = "1.9.0"
```

Comment:
what happened to 1.8.0?

Reply:
We didn't merge in upstream 1.8.0 so we're jumping two minor bumps here 😄